v2/datastream-to-spanner/src/test/resources/session-file-sharded.json

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
Is it a good idea to keep an inconsistent session file? t1 table does not have the shardedColId field.